client/service/src/builder.rs

```diff
@@ -916,23 +921,23 @@ ServiceBuilder<
 			let is_validator = config.roles.is_authority();
 
 			let (import_stream, finality_stream) = (
-				client.import_notification_stream().map(|n| ChainEvent::NewBlock {
-					id: BlockId::Hash(n.hash),
-					header: n.header,
-					retracted: n.retracted,
-					is_new_best: n.is_new_best,
-				}),
+				client.all_blocks_notification_stream()
```

Comment:
I am worried that when we are syncing >1k blk/s that this channel will clog up or become a bottleneck. Either that or it will race against state pruning.

Can you elaborate a little bit on the use case for running offchain workers on every block?

Reply:
The initial idea was that for an import of all blocks and writing to the offchain worker db, we need a stream that also covers the initial sync - which to this point none did (there is an if case which excludes the initial import from being triggered for sync messages).

Reply:
This queue was removed again